mipengine/node_tasks_DTOs.py

```diff
@@ -170,28 +225,21 @@ class NodeLiteralDTO(NodeUDFDTO):
 
 class NodeTableDTO(NodeUDFDTO):
     type = _NodeUDFDTOType.TABLE
-    value: str
-
-
-class NodeSMPCValueDTO(ImmutableBaseModel):
-    template: NodeTableDTO
-    sum_op_values: NodeTableDTO = None
-    min_op_values: NodeTableDTO = None
-    max_op_values: NodeTableDTO = None
+    value: TableInfo
 
 
 class NodeSMPCDTO(NodeUDFDTO):
     type = _NodeUDFDTOType.SMPC
-    value: NodeSMPCValueDTO
+    value: SMPCTablesInfo
 
 
-class UDFPosArguments(ImmutableBaseModel):
+class NodeUDFPosArguments(ImmutableBaseModel):
     args: List[Union[NodeLiteralDTO, NodeTableDTO, NodeSMPCDTO]]
 
 
-class UDFKeyArguments(ImmutableBaseModel):
+class NodeUDFKeyArguments(ImmutableBaseModel):
     args: Dict[str, Union[NodeLiteralDTO, NodeTableDTO, NodeSMPCDTO]]
 
 
-class UDFResults(ImmutableBaseModel):
-    results: List[Union[NodeTableDTO, NodeSMPCDTO]]
+class NodeUDFResults(ImmutableBaseModel):
```

Comment:
Then the disambiguation between the two should happen where the ambiguity arises, which is inside the `./mipengine/node/tasks/udfs.py` module by doing an `import as`
```
from mipengine.node_tasks_DTOs import UDFResults as NodeUDFResults
```
without burdening other modules with a disambiguation that is irrelevant to them.
..that is though if this is the only reason for that

Reply:
This is an option that we could follow yes, but this, from my perspective, creates another ambiguity where the same thing is different between different modules. When I inspect these methods for example I will see `NodeUDFResults` as a parameter or as a return type, but this object won't exist and I will have to search what that object is actually named.

I understand that this looks like a "burden" to other modules, especially to the controller that has no knowledge of the NODE internals, but for someone working on both the NODE and the CONTROLLER, it's problematic to have to think each time, where you are and what you are importing exactly.

That's why I thought that it would be better to explicitly differentiate them and solve this confusion for good.

Reply:
again, it is not obvious why a class inside a module called `node_tasks_DTOs` needs to have a prefix "Node" in each name. One would anyway expect to find classes that are only related to "node tasks" in that module. So you are saying that I have to have the "Node" prefix in the names of this module because there is another module, that is completely independent from this module, that has a class with similar name/functionality and this is a problem that has to be dealt with    
in this module, although the 2 modules should be completely independent.